Magic/SpellSelectorVM.cs

```diff
@@ -51,13 +52,23 @@ public void OnToggle(bool open)
                         OnItemHovered
                     )
                 );
-                SpellManager
-                    .GetAllSpell()
-                    .ForEach(spell =>
-                        Spells.Add(
-                            new EquipmentActionItemVM(spell.Name, spell.Icon, spell, OnItemHovered)
-                        )
-                    );
+
+                var agent = Agent.Main?.Character as CharacterObject;
```

Comment:
Probably meant to do 

```suggestion
                var character = Agent.Main?.Character as CharacterObject;
```